zhaquirks/tuya/__init__.py

```diff
@@ -1488,6 +1489,9 @@ class TuyaNewManufCluster(CustomCluster):
     ep_attribute: str = "tuya_manufacturer"
 
     server_commands = {
+        TUYA_QUERY_DATA: foundation.ZCLCommandDef(
+            "query_data", {}, False, is_manufacturer_specific=True
```

Comment:
There's no `data` structure for this command?
Can you point us to any command message or use case?

Reply:
There are Tuya devices that will not start nor send information w/o DATA_QUERY messages. For these, a DATA_QUERY package has to be sent on startup and also later for polling: no information is sent except if a DATA_QUERY packet is sent. 

See the forthcoming [zha-quirk](https://gist.github.com/tschiex/28237907a4f0c569015b1869c8ffe66a) for an example of usage in ZHA. I followed the suggestion of TheJulianJES here. 

See also the corresponding code in Z2MQTT [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/a60d4f6e3b709160a045e836200910a1ab8660dc/src/lib/tuya.ts#L77)